### PR TITLE
Shelter agency user can decline on steps 3, 4, and 5 of the default r…

### PR DESCRIPTION
**Pull request by eanders**
…oute, admins can approve/reject declines

### CODE REVIEW

File: app/controllers/match_decisions_controller.rb [hunk 1]
```diff
@@ -137,86 +133,83 @@ def update
...
     end
+    redirect_to access_context.match_decision_path(@match, @decision, redirect: 'true')
   end
 
   private
 
-    def find_match!
-      @match = match_scope.find params[:match_id]
-    end
+  def find_match!
```
Comment (awisema) [inline app/controllers/match_decisions_controller.rb]: In addition to being un-indented, these should get their own `private` declaration/

File: app/models/match_decisions/approve_match_housing_subsidy_admin.rb [hunk 2]
```diff
@@ -121,15 +139,19 @@ def canceled
...
+        end
+      end
     end
     private_constant :StatusCallbacks
 
     private def note_present_if_status_declined
-      if note.blank? && status == 'declined'
-        errors.add :note, 'Please note why the match is declined.'
-      end
+      errors.add :note, 'Please note why the match is declined.' if note.blank? && status == 'declined'
```
Comment (awisema) [inline app/models/match_decisions/approve_match_housing_subsidy_admin.rb]: Should this also be true for `shelter_declined`?

File: app/models/match_decisions/five/five_screening.rb [hunk 3]
```diff
@@ -36,8 +36,8 @@ def statuses
       {
         pending: 'Pending',
         expiration_update: 'Pending',
-        mitigation_required: "#{_('Mitigation required')}",
-        mitigation_not_required: "#{_('No mitigation required')}",
+        mitigation_required: _('Mitigation required').to_s,
+        mitigation_not_required: _('No mitigation required').to_s,
```
Comment (awisema) [inline app/models/match_decisions/five/five_screening.rb]: I think the `to_s` is redundant?

File: docker/docker-compose.yml [hunk 4]
```diff
@@ -21,6 +21,7 @@ x-app: &app
     PSQL_HISTFILE: /app/log/.psql_history
     EDITOR: vi
     CHROMEDRIVER_VERSION: 79.0.3945.36
+    BUNDLE_FORCE_RUBY_PLATFORM: true
```
Comment (awisema) [inline docker/docker-compose.yml:25]: This line blows up for me -- something about how true isn't a valid type for a docker environment variable. Does it not do it for you?

**Review by awisema — APPROVED** [review]

File: docker/docker-compose.yml (hunk 4 above)
Comment (eanders) [reply docker/docker-compose.yml:25]: It works for me, but I did have to `docker-compose build`.  Let me know if that doesn't fix it for you, I have a less good solution we can try.

File: app/models/match_decisions/approve_match_housing_subsidy_admin.rb (hunk 2 above)
Comment (eanders) [reply app/models/match_decisions/approve_match_housing_subsidy_admin.rb]: This doesn't appear to be connected as one would expect.  Declines don't actually have a place to input a note.  But it works as expected.

**Pull request merged**